Python/Product/PythonTools/PythonTools/Debugger/DebuggerHelper.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PythonTools.Debugger.DebugEngine;
+using Microsoft.PythonTools.Infrastructure;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Debugger.Interop;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.PythonTools.Debugger {
+    /// <summary>
+    /// Helper class to expose IDebug* variables for use elsewhere (instead of using EnvDTE.Debugger)
+    /// </summary>
+    public class DebuggerHelper: IDebugEventCallback2 {
```

Comment:
This entire file should go away once I remove the debug interactive repl. It reimplements all of the PythonProcess/Thread stuff.